common/src/main/java/dev/latvian/mods/kubejs/block/PickBlockCallbackJS.java

```diff
@@ -0,0 +1,36 @@
+package dev.latvian.mods.kubejs.block;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.block.state.BlockState;
+
+public class PickBlockCallbackJS {
```

Comment:
Why is there no way to get a BlockContainerJS from here? Typically these callbacks provide one of those rather than the individual components that make it up such as BlockGetter, BlockState and BlockPos. Have a look at the other existing callbacks.

Reply:
Alright, will do that.